repos/fdbt-site/src/pages/service.tsx

```diff
@@ -24,7 +24,7 @@ const errorId = 'service';
 interface ServiceProps {
     operator: string;
     passengerType: string;
-    services: ServiceType[];
```

Comment:
I dont think we need to change this type to be MyFaresService - just need to add endDate to ServiceType